README.md

```diff
@@ -601,6 +601,12 @@ golang:
    # if unset this defaults to $GONOPROXY
    # SYFT_GOLANG_NOPROXY env var
    no-proxy: ""
+  
+java:
+   # when running across pom.xml files that could have more information
```

Comment:
it's probably worth noting in this comment that this is particularly useful in situations where the parent pom has this information but it is not accessible from within the final built artifact.

Reply:
updated readme